04_pig_latin/pig_latin.rb

```diff
@@ -1 +1,12 @@
 #write your code here
+def translate (word)
+    vowels = %w{a e i o u}
+    word.gsub(/(\A|\s)\w+/) do |str|
```

Comment:
Good use of regex here. In case you don't know about this tool, it is one I use all the time to test my regex and thought I'd share: https://rubular.com/